cocoa/src/toga_cocoa/app.py

```diff
@@ -400,6 +400,18 @@ def exit_full_screen(self, windows):
             window.content._impl.native.exitFullScreenModeWithOptions(opts)
             window.content.refresh()
 
+    def get_cursor_position(self):
+        self.interface.factory.not_implemented("App.get_cursor_position")
+
+    def set_cursor_position(self, cursor_position):
+        self.interface.factory.not_implemented("App.set_cursor_position")
+
+    def is_cursor_visible(self):
+        return self._cursor_visible
```

Comment:
This will raise an error. It should log a not_implemented, and return True (which is a reasonable default, given there's no `set_cursor_visible` implementation.

Reply:
`_cursor_visible` was already defined in cocoa, and it is used by `show_cursor()` and `hide_cursor()` in cocoa. So, it shouldn't error out. I have modified to log not_implemented and return True on other platforms.